recipes/fred/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "fred" %}
+{% set version = "2.0.2" %}
+{% set sha256 = "223f139b35357e25d98644eb2668b9140f1489f4befd1e337a83fa6253cc0d54" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/FRED-2/Fred2/archive/{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  noarch: python
+  number: 0
+  skip: True  # [not py27]
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - pyomo
+    - pandas
+    - pyvcf
+    - biopython
+    - mysql-python
+  run:
+    - python
+    - setuptools
+    - pyomo
+    - pandas
+    - pyvcf
+    - biopython
+    - mysql-python 
+
+test:
+  commands:
+    - python -c "from Fred2.Core import Allele, Peptide, Protein,generate_peptides_from_proteins"
+
+about:
+  home: http://nf-co.re/
+  license: MIT
+  summary: 'Python package with helper tools for the nf-core community.'
+
+extra:
+  container:
+    # click requires a unicode locale when used with Python 3
```

Comment:
But you are not using click here isn't it?

Reply:
Whoops, thats a relict of my template. I'll remove it.

Reply:
Changed it, renamed the package as well to fred2